src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaControllerTest.php

```diff
@@ -116,24 +118,49 @@ protected function setUpMedia()
         $tag2->setChanged(new DateTime());
         $tag2->setName('Tag 2');
 
+        $this->em->persist($tag1);
+        $this->em->persist($tag2);
+        $this->em->persist($this->documentType);
+        $this->em->persist($this->imageType);
+        $this->em->persist($this->videoType);
+
+        $this->em->flush();
+    }
+    
+    protected function createMedia($name)
+    {
+        $media = new Media();
+        $media->setCreated(new DateTime());
+        $media->setChanged(new DateTime());
+        $media->setType($this->imageType);
+
+        // create file
+        $file = new File();
+        $file->setVersion(1);
```

Comment:
Not part of this PR, but we could probably handle this with an event listener/subscriber.